src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -100,6 +100,14 @@
     "message": "Count of EC2 instances backing the cluster compute fleet",
     "event_type": "cluster-instance-count",
 }
+SCONTROL_SHOW_JOB_INFORMATION = {
+    "message": "Job information from scontrol",
+    "event_type": "scontrol-show-job-information",
```

Comment:
I'm not sure we should include `scontrol` in the event name - maybe something more generic like just "job-information" or "scheduler-job-information"?

Reply:
The reason is that now there's another cluster SlurmJob that represent the jobs information from slurm resume for job level based scaling. In case there will be json log passing from there, I rename the jobs info from scontrol shows jobs output to avoid confusion